routes/api/quiz/controller.js

```diff
@@ -33,7 +33,7 @@ class QuizController extends BaseController {
         model: DB.questions,
         include: {
           model: DB.choices,
-          attributes: ['id', 'title', 'positiveWeight', 'negativeWeight']
+          attributes: ['id', 'title', 'positiveWeight', 'negativeWeight, multipleCorrect']
```

Comment:
Why?

Reply:
to extract the multiple correct attribute from question